### PR TITLE
feat(Updater/Admin): Clarify instructions for `upgrade.disable-web` to avoid confusion/reduce help requests/help downstream Docker image maintainers 

### PR DESCRIPTION
**Pull request by joshtrichards**
## Summary

In Docker environments the web-updater is not used, but when the `upgrade.disable-web` parameter is used to attempt to deter people from using or clicking the updater button, the instructions are still unclear and - worse - misleading.

When using this config option, the instructions merely say "use the command line Updater" but that isn't correct in many environments (think downstream Docker image maintainers):

image

Before: "Please use the command line updater to update. "

After: "Web updater is disabled. Please use the command line updater or the appropriate update mechanism for your installation method (e.g. Docker pull) to update."

P.S. The Download button remaining is also confusing, but that's existing behavior and may be applicable in some scenarios admittedly... so that can be left for a future follow-up.

## TODO

- [x] Get reviewed

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by susnux — APPROVED** [review]

**Review by emoral435 — APPROVED** [review]
Yeah, I agree with the previous message likely leading to confusion - this new message is good!

File: apps/updatenotification/src/components/UpdateNotification.vue [hunk 1]
```diff
@@ -56,7 +56,7 @@
 						class="button"
 						:class="{ hidden: !updaterEnabled }">{{ t('updatenotification', 'Download now') }}</a>
 					<span v-if="updaterEnabled && !webUpdaterEnabled">
-						{{ t('updatenotification', 'Please use the command line updater to update.') }}
+						{{ t('updatenotification', 'Web updater is disabled. Please use the command line updater or the appropriate update mechanism for your installation method (e.g. Docker pull) to update.') }}
```
Comment (kesselb) [inline apps/updatenotification/src/components/UpdateNotification.vue:59]: For some reason, I would prefer "docker pull" over "Docker pull". 
However, we have two approvals and a green CI so let's do it ;)

**Comment by kesselb** [comment]
Thank you @joshtrichards :pray:

**Pull request merged**